services/app-api/handlers/reports/update.ts

```diff
@@ -42,12 +42,12 @@ export const updateReport = handler(async (event, context) => {
     // validate report field data
     const currentReport = JSON.parse(getCurrentReport.body);
     const { formTemplate } = currentReport;
-    const validatedFieldData = validateFieldData(
+    const validatedFieldData = await validateFieldData(
       formTemplate.validationJson,
       unvalidatedFieldData
     );
 
-    if (validatedMetadata && validatedFieldData) {
+    if (validatedMetadata) {
```

Comment:
I might be missing something here, but why remove the check for validatedFieldData?

Reply:
Good call. Added an await to the above function and now it's covered - will add it back!